unity_jobs_simulator/Assets/Scenes/ur10eScene.unity

```diff
@@ -0,0 +1,777 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Angle sensitivity in the scene is 0, so the robot isn't reaching the goal position.